File: SDKs/amazonaws.com/cloudfront/2020-05-31/typescript/src/sdk/models/shared/cookienames.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// CookieNames
/** 
 * Contains a list of cookie names.
**/
export class CookieNames extends SpeakeasyBase {
  @SpeakeasyMetadata()
  items?: Record<string, any>[];

  @SpeakeasyMetadata()
  quantity: number;
}

File: SDKs/amazonaws.com/cloudfront/2020-05-31/python/src/sdk/models/shared/cookienames.py
import dataclasses
from typing import Any,Optional


@dataclasses.dataclass
class CookieNames:
    r"""CookieNames
    Contains a list of cookie names.
    """
    
    quantity: int = dataclasses.field()
    items: Optional[list[dict[str, Any]]] = dataclasses.field(default=None)
    
